tooling/miniutil/src/build/expr.rs

```diff
@@ -189,6 +189,9 @@ pub fn bool_or(l: ValueExpr, r: ValueExpr) -> ValueExpr {
 pub fn bool_xor(l: ValueExpr, r: ValueExpr) -> ValueExpr {
     bool_binop(BoolBinOp::BitXor, l, r)
 }
+pub fn bool_eq(l: ValueExpr, r: ValueExpr) -> ValueExpr {
```

Comment:
I don't quite see the point of this function... it uses a different encoding than minimize, and it's not even used by anything?

Reply:
I thought it can be useful when writing programs with miniutil. I removed it.